src/index.html

```diff
@@ -17,16 +17,48 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search search--big">
+      <form
+        class="search__form"
+        data-qa="big"
+      >
+        <div class="search__container search__container--big">
+          <span class="search__icon search__icon--big">
+            <img
+              src="images/Search.svg"
+              alt="Search icon"
+            />
```

Comment:
The icon should be background 
<img width="549" alt="image" src="https://github.com/user-attachments/assets/f09ce220-cb58-4224-b2bf-011a04160f2a" />